xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-api/src/main/java/org/xwiki/livedata/internal/LiveDataRendererParameters.java

```diff
@@ -0,0 +1,258 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.livedata.internal;
+
+import org.xwiki.properties.annotation.PropertyDescription;
+
+/**
+ * Parameters for {@link LiveDataRenderer}.
+ *
+ * @version $Id$
+ * @since 16.0.0RC1
+ */
+public class LiveDataRendererParameters
+{
+    private String id;
+
+    private String properties;
+
+    private String source;
+
+    private String sourceParameters;
+
+    private String sort;
+
+    private String filters;
+
+    private Integer limit;
+
+    private Long offset;
+
+    private String layouts;
+
+    private Boolean showPageSizeDropdown;
+
+    private String pageSizes;
+
+    /**
+     * @return the live data instance id
+     */
+    public String getId()
+    {
+        return this.id;
+    }
+
+    /**
+     * Sets the live data instance id.
+     *
+     * @param id the live data instance id
+     */
+    @PropertyDescription("The live data instance id.")
```

Comment:
Is this a leftover ?

If not, then I don't understand why there and not on the other setters.

Reply:
Yes, that a miss. Thanks.
@tmortagne do you agree with the revapi exclude https://github.com/xwiki/xwiki-platform/pull/2327/files/46bda7abc69c944390012b9fc5072261a34d2e20#diff-2f81214bd08060b9f2f1bdd7cb0dda0e7109ccadbf852cda6ddb864abc29c8e3R196-R208 ?

Reply:
Yes, according to Revapi's own documentation, it's not a runtime breakage. It's only a "potentially breaking", it report it just in case basically.